internal/signalfx-agent/bundle/collectd-plugins.yaml

```diff
@@ -31,7 +31,7 @@
   version: v1.3.7
   repo: signalfx/collectd-mongodb
   pip_packages:
-    - pymongo[tls]==3.13.0
+    - pymongo[tls]==4.6.3
```

Comment:
This major version update might be breaking for the collectd-mongodb plugin. For e.g., [v4.x replaces the parameter names for ssl args](https://pymongo.readthedocs.io/en/stable/migrate-to-pymongo4.html#renamed-uri-options) which we seem to be using in [our plugin](https://github.com/signalfx/collectd-mongodb/blob/master/mongodb.py#L88-L95).

Reply:
ah, I assumed we had an integration test covering this.

Reply:
Good catch! I'll look into this 👍